packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx

```diff
@@ -60,10 +69,19 @@ export const IntercomCard: React.FC<
     telegramId,
   } = useNotifiSubscriptionContext();
 
+  const alertName = 'NOTIFI_CHAT_MESSAGES';
+
+  useEffect(() => {
+    if (loading) {
+      return;
+    }
+    const hasAlert = alerts[alertName] !== undefined;
+    setHasChatAlert(hasAlert);
+  }, [loading, alerts]);
+
   const hasErrors = emailErrorMessage !== '' || smsErrorMessage !== '';
   const disabled =
-    (email === '' && phoneNumber === '' && telegramId === '' && !checked) ||
-    hasErrors;
+    (email === '' && phoneNumber === '' && telegramId === '') || hasErrors;
```

Comment:
use setTimeout to set chatAlertErrorMessages to empty string after 5 seconds here, and for the target input error(email, sms, telegram), the error message goes away when user focus on the input field.

Reply:
Great, that works well! Long-term, we can bring in `useErrorHandler` hook from client which has a queue functions within instead of using `setTimeout`. (Some devs don't like `setTimeout` but I don't mind it for a use case like this).

Reply:
agreed, and I'll add a todo here for future refactor.